modules/private-dns-resolver/README.md

```diff
@@ -0,0 +1,31 @@
+# Private DNS Resolver
+
+This module creates a [Private DNS Resolver](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/private_dns_resolver) and associated [Diagnostic Setting](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/monitor_diagnostic_setting).
```

Comment:
No diag settings, and don't think they're avail for this resource, however think we should mention endpoints.
```suggestion
This module creates a [Private DNS Resolver](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/private_dns_resolver), including [Inbound](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/private_dns_resolver_inbound_endpoint) and [Outbound](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/private_dns_resolver_outbound_endpoint) Endpoints.
```